desktop-app/app/components/ExtensionsManager/index.js

```diff
@@ -58,13 +59,19 @@ export default function ExtensionsManager({triggerNavigationReload}) {
     setLoading(true);
     setErrorMessage('');
 
-    const id = extensionId
-      .replace(/\/$/, '')
-      .split('/')
-      .pop();
+    const isLocalExtension = fs.statSync(extensionId).isDirectory();
```

Comment:
@prashantpalikhe Can you please move this logic to the `main` process? We should try and keep `node` usage as less as possible in the `renderer`.

Reply:
Updated. Can you check?